### DISCUSSION TITLE
AWS EC2 install with dynamic IP

### DISCUSSION THREAD

**Discussion by danrossi** [OvenMediaLabs/OvenMediaEngine#1645] [Help]
I had a look at these docs to setup OME in EC2. However is there a way to relaunch the current container if the dynamic ip changes ? reserving static ip costs money and will keep being charged when the instance is off. Is it as simple as updating an environment variable for the host ip ? 

https://medium.com/@AirenSoft/ovenmediaengine-getting-started-with-ome-docker-launcher-2d1723279b1c

**Comment by getroot** [comment]
As far as I know, EC2 doesn't change IP while the instance is on. IP can only change when the instance is restarted.

**Reply by danrossi** [reply]
It seems when restarting I just update the environment command when starting the container. It seems to be ok with dynamic ip.
